docker/linux/unittest/Dockerfile

```diff
@@ -87,6 +87,29 @@ RUN cd /tmp \
     && echo ${COBALT_CLANG_VER} >> ${COBALT_CLANG_TC_HOME}/cr_build_revision \
     && rm llvm-code-coverage-llvmorg-${COBALT_CLANG_VER}.tgz
 
+# === Install Clang 16 toolchain for llvm-symbolization.
+ARG CLANG_16_VER
```

Comment:
I'm not sure this is even needed tbh. We don't run unit tests in internal buildbot at all. Only the build container should need this

Reply:
done.